src/twisted/python/failure.py

```diff
@@ -419,23 +393,6 @@ def _extrapolate(self, otherFailure):
         # Copy all infos from that failure (including self.frames).
```

Comment:
Nit: update this comment?

```suggestion
        # Copy all infos from that failure (including self._frames).
```